src/shared/components/Layout/FiltersContainer/FilterContainer.styled.tsx

```diff
@@ -0,0 +1,47 @@
+import { Button, styled } from "@mui/material";
+import heroImageUrl from "/src/shared/assets/hero.svg";
+
+export const FilterContainerStyled = styled("div")`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 100vw;
```

Comment:
I suggest to use `vw` and `vh` only if you absolutely have to. Can you use `100%` instead?